src/main/java/edu/ksu/canvas/attendance/controller/CourseConfigurationController.java

```diff
@@ -1,6 +1,7 @@
 package edu.ksu.canvas.attendance.controller;
 
 
+import com.sun.org.apache.xpath.internal.operations.Mod;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1191)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.sun.org.apache.xpath.internal.operations.Mod'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)